Update update-phone-number-and-email-address.md (#3203)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/update-phone-number-and-email-address.md

  
If you're looking for how to change your company's profile information, such as company name and address, company phone number, and technical contact information, see [Change your organization's address, technical contact email, and other information](change-address-contact-and-more.md).
  

> ## To update your phone number, and email address

Reviewer: ```suggestion
## To update your phone number and email address
```

> Use the **Security Info** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 

Reviewer: ```suggestion
Use the **Security Info** page to change your mobile phone number and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
```

  
1. Browse to the Microsoft 365 admin center.

2. In the header, select your profile icon \> **My account** \> **Security Info**.

> 3. In the **Security info** tab, select **Add Method** \> **Phone** \> **Alternate Phone** or **Email** to add details. To update your mobile, phone, and alternate email address details, select **Change**. Make sure you use something different than your Microsoft email address for your alternate email address.

Reviewer: ```suggestion
3. In the **Security info** tab, select **Add Method** \> **Phone** \> **Alternate Phone** or **Email** to add details. To update your mobile, phone, and alternate email address details, select **Change**. Make sure you use something other than your Microsoft email address for your alternate email address.
```

> 3. In the **Security info** tab Click **Add Method**, select **Phone**, **Alternate Phone**, Or **Email** to add details. To Update your mobile, phone, and alternate email address you can edit them by clicking on **Change**. Make sure you use something different from your Microsoft email address for your alternate email address.

Reviewer: ```suggestion
3. In the **Security info** tab, select **Add Method** \> **Phone** \> **Alternate Phone** or **Email** to add details. To update your mobile, phone, and alternate email address details, select **Change**. Make sure you use something different than your Microsoft email address for your alternate email address.
```
